apps/src/templates/InlineDropdownMenu.jsx

```diff
@@ -0,0 +1,143 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+import onClickOutside from 'react-onclickoutside';
+
+// HTML Adapted from:
+// https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-actions.html
+
+// Displays an accessible dropdown menu with actions or links
+// TODO: Convert to a functional component once Radium is upgraded past 0.26.0
+// Our version of Radium doesn't support useState, but it has been patched in
+// later versions
+export const InlineDropdownMenu = class InlineDropdownMenuComponent extends Component {
+  static propTypes = {
+    icon: PropTypes.string.isRequired,
+    children: props => {
+      React.Children.map(props.children, child => {
```

Comment:
yay for complex propTypes for extra validation!

Reply:
🎉